### PR TITLE
Assorted data pipeline changes

### PR DESCRIPTION
**Pull request by wdwatkins**
Description
-----------
Re: data issues from @amrhoades post in Teams
	
- [ ] Use water year instead of calendar year  ​​
- [x] Change active urban/non-urban calculation to slider focus year (2018) instead of 2019 . This is the urban % calc for the nation  ​
- [ ] Use list of gages funded in 2019 to populate the missing CA gages  ​
- [x] Data pull: 100 year gages​ .csv @collnell

### CODE REVIEW

File: data_processing_pipeline/2_process/src/process_urban_areas.R [hunk 1]
```diff
@@ -9,7 +9,7 @@ process_digitized_urban_extent <- function(target_name, digitized_shp_fn, proj_s
 process_modern_urban_extent <- function(target_name, urban_extent_shp_fn, state_nm, proj_str) {
   state_sf <- st_as_sf(maps::map("state", state_nm, plot = FALSE, fill = TRUE)) %>% 
     st_transform(crs = proj_str) %>%
-    lwgeom::st_make_valid() %>% 
+    st_make_valid() %>%
```
Comment (wdwatkins) [inline data_processing_pipeline/2_process/src/process_urban_areas.R:12]: st_make_valid is an `sf` function

Comment (lindsayplatt) [reply data_processing_pipeline/2_process/src/process_urban_areas.R:12]: is it? it only works for me from `lwgeom`

**Comment by lindsayplatt** [comment]
Actually, going to close this without merging. Colleen ended up needing the full dataset not just hundred year gages and I don't want to confuse our future selves by having this in there. I've got the other changes (2019-->2018, ~lwgeom~) locally and those will come with my PR for using water year instead of calendar year.

**Comment by lindsayplatt** [comment]
I am using scipiper version `0.0.20` and I did not need to add `include: 1_fetch.yml` to `2_process.yml` to build the pipeline using `scmake()`

**Pull request closed without merging**